torch_geometric/nn/conv/message_passing.py

```diff
@@ -115,8 +116,12 @@ def __init__(self, aggr: Optional[str] = "add",
         # Hooks:
         self._propagate_forward_pre_hooks = OrderedDict()
         self._propagate_forward_hooks = OrderedDict()
+        self._edges_propagate_forward_pre_hooks = OrderedDict()
```

Comment:
Do you think we need two hooks here? In contrast to `propagate`, edge updates only call a single function.

Reply:
Good point, this seems unneeded, I will change to one.